TESTS/usb_device/msd/README.md

```diff
@@ -10,3 +10,12 @@ Test run command:
 ```bash
 mbed test -t COMPILER -m TARGET -n tests-usb_device-msd
 ```
+
+
+## Windows 8/10:
+By default Windows 8 and 10 access and write to removable drives shortly after they are connected. It's caused by drive indexing mechanisms. Because disk used in test has only 64kB its content can be easily corrupted by writing large files, what actually was encountered during test runs.
```

Comment:
NOTE: If #10688 comes first, please move this info to the [known-issues section](https://github.com/ARMmbed/mbed-os/blob/0ae1be0e6588b07bf3a7d242937ab4a224ff2883/TESTS/usb_device/README.md#known-issues) of the top-level `README` file.

Reply:
Rebased
Changes moved from `TESTS/usb_device/msd/README.md` to `TESTS/usb_device/README.md`